packages/plugin-electron-session/session.js

```diff
@@ -11,9 +11,8 @@ module.exports = (app, BrowserWindow, filestore) => ({
       return
     }
 
-    // jump through hoops to ensure the session gets started _after_ we have a device id
-    Promise.all([filestore.getDeviceInfo(), app.whenReady()])
-      .then(() => { setTimeout(() => client.startSession(), 10) })
+    app.whenReady()
+      .then(() => client.startSession())
```

Comment:
Calling `getDeviceInfo()` would incur another unnecessary sync fs call, so I want to avoid that if possible. Since `app.whenReady()` is async it's (virtually) guaranteed to happen after the sync file read.

Reply:
The "virtually" is making me nervous 😀